corehq/apps/api/resources/v0_5.py

```diff
@@ -1086,6 +1087,10 @@ def to_obj(action_times):
         '''
         return namedtuple('action_times', list(action_times))(**action_times)
 
+    def dehydrate(self, bundle):
+        bundle.data['user_id'] = get_user_id_by_username(bundle.data['user'])
```

Comment:
no it shouldn't be added to the fields of the class. 

> how are those used? In the internal step that turns an object into a "bundle"?

From my understanding of it , yes. Tastypie will use the field definitions to determine how to extract the values of local_date, UTC_start_time, UTC_end_time, and user from the object that's returned from `obj_get_list` and include them in the bundle. The object doesn't have a `user_id `  field so it would give a `empty attribute error ` if we add `user_ids `to the fields on the class